tests/child-test.js

```diff
@@ -0,0 +1,23 @@
+var spawn = require('child_process').spawn;
+
+test('simple seg fault', function (t) {
+   
+   var seg   = spawn('node', ['example.js']);
```

Comment:
Maybe call this "child".

Also, example.js is just a basic demo.  It might be worth adding dedicated test code that runs in the child process.